vipps-recurring-api.md

```diff
@@ -1157,14 +1182,13 @@ GET charge response where amount is higher than the users max amount:
     "transactionId": null,
     "description": "Monthly payment",
     "type": "RECURRING",
-    "failureReason": "charge_amount_too_high",
-    "failureDescription": "Amount is higher than the users specified max amount"
```

Comment:
As noted, we will keep failureReason for some cases, like this, because it makes sense to let the merchant know this.

Reply:
I think we will use an event log.